pkg/store/bucket.go

```diff
@@ -2382,8 +2382,10 @@ type bucketIndexReader struct {
 	dec   *index.Decoder
 	stats *queryStats
 
-	mtx          sync.Mutex
-	loadedSeries map[storage.SeriesRef][]byte
+	mtx sync.Mutex
+	// Separate mutex to protect loadedSeries.
```

Comment:
I think it's obvious, I suggest removing this comment.

Reply:
Made the change.